Philips.CodeAnalysis.Test/Maintainability/Maintainability/AvoidThrowingUnexpectedExceptionsAnalyzerTest.cs

```diff
@@ -173,20 +173,20 @@ public async Task CorrectCodeShouldNotTriggerAnyDiagnosticsAsync(string testCode
 		}
 
 		[DataTestMethod]
-		[DataRow(WrongStaticConstructor, DisplayName = nameof(WrongStaticConstructor)),
```

Comment:
Assume these non-doc changes were intentional?

Reply:
Yes, this one solved a bug in the Message text. Updated the tests accordingly